application_test.go

```diff
@@ -65,6 +65,12 @@ func minimalApplicationMap() map[interface{}]interface{} {
 				minimalUnitMap(),
 			},
 		},
+		"offers": map[interface{}]interface{}{
```

Comment:
This isn't really minimal. Probably not needed here.

Is there a complete application?

Reply:
I opted to extend the minimal application so I can get the test-suite to exercise the offer code in a lot more scenarios (plus, the base class also auto-generates serialization/deserialization tests for applications so I go that tested with my changes for free).

Having said that, your argument does make sense to me. As there is not a complete application block defined in the test suite, I have pushed https://github.com/juju/description/pull/52/commits/fadfc38db7f532900f129e3cde1181cc34bdc86a to extract the offer-bits out of the minimal application and test the offer-related application bits in isolation.